web/src/classic/components/organisms/Settings/SettingPage/hooks.ts

```diff
@@ -84,6 +84,7 @@ export default (params: Params) => {
               name: project.name,
               isArchived: project.isArchived,
               sceneId: project.scene?.id,
+              projectType: project.coreSupport ? "beta" : "classic" ?? "classic",
```

Comment:
Can you leave it as `coreSupport`? The reason we added coreSupport and not something referencing `beta` or `classic` is so its more meaningful after we remove classic support. Also, as beta becomes more stable we might rename it to something else and make it available to prod users, at which point we would have to update all references to beta

Reply:
would it be difficult to anyone read the code to understand what coreSupport means if we remove this ?

Reply:
For now, just leave this. But you don't need the final `?? "classic"` because `project.coreSupport ? "beta" : "classic"` will always give a value (beta or classic). It'll never be undefined 👍🏻